js/ui/scroll_view/ui.scrollable.js

```diff
@@ -592,6 +592,7 @@ var Scrollable = DOMComponent.inherit({
     /**
     * @name dxscrollablemethods_update
     * @publicName update()
+    * @return Promise<void>
```

Comment:
think just Promise should be here
Am I right @lukyanovas

Reply:
It doesn't matter